eventhub/Module.md

```diff
@@ -102,14 +80,14 @@ URI to the event hub namespace.
 Note:
 You must specify the SAS key name, SAS key and the resource URI when configuring the Azure Event Hub Client connector.
 
-### Step 3: Specify the (Optional) Broker Properties and User Properties
+#### Step 3: Specify the (Optional) Broker Properties and User Properties
```

Comment:
the step is optional? or is there a optional broker?

Reply:
Its an optional parameter. Fixed in df7c75b . Thank you.